setup.py

```diff
@@ -26,8 +26,8 @@ def read(*rnames):
         return f.read()
 
 setup(name='zope.size',
-      version='4.2.0.dev0',
-      url='http://pypi.python.org/pypi/zope.size',
+      version=read('version.txt'),
+      url='http://zopesize.readthedocs.io',
```

Comment:
Cool, I'm going to cite this comment as evidence over on https://github.com/zopefoundation/zope.container/pull/8#discussion_r92415358 😄